packaging/rpm/mock-on-docker.sh

```diff
@@ -10,8 +10,8 @@
 
 set -ex
 
-_DOCKER_IMAGE=fedora:33
-_MOCK_CONFIGS="epel-7-x86_64 epel-8-x86_64"
+_DOCKER_IMAGE=fedora:35
+_MOCK_CONFIGS="centos+epel-7-x86_64 centos-stream+epel-8-x86_64"
```

Comment:
Can you also change the centos:8 image in test-on-docker.sh to `redhat/ubi8:8.5-226` ?